ocs_ci/ocs/longevity.py

```diff
@@ -554,6 +722,319 @@ def stage_0(
 
         return kube_job_file_list
 
+    def stage_4(
+        self,
+        project_factory,
+        multi_pvc_factory,
+        pod_factory,
+        multi_pvc_clone_factory,
+        multi_snapshot_factory,
+        snapshot_restore_factory,
+        teardown_factory,
+        num_of_pvcs=30,
+        pvc_size=2,
+        run_time=1440,
+        pvc_size_new=4,
```

Comment:
I think it will be better if we can allow users to choose how much amount of data they wanted to fill in perc,

Reply:
Ack, will add that.